### PR TITLE
Elected candidates should be at the top of the list

### PR DESCRIPTION
**Pull request by VirginiaDooley**
Closes https://github.com/DemocracyClub/WhoCanIVoteFor/issues/1708

Screenshot 2024-03-21 at 2 04 06 PM

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206889313171652

### CODE REVIEW

**Comment by VirginiaDooley** [comment]
> This should already be done in [this block](https://github.com/DemocracyClub/WhoCanIVoteFor/blob/73623a44ac32b2ace5a4bdbac8b740c4df235a05/wcivf/apps/elections/views/mixins.py#L98-L108). Can you take a look at why it might not be working?

The following block that sorts by `votes_cast` is the culprit. I've made a change to sort by elected then votes cast in the same step and it seems to work as expected.

File: wcivf/apps/elections/views/mixins.py [hunk 1]
```diff
@@ -100,12 +96,8 @@ def people_for_ballot(self, postelection, compact=False):
         else:
             order_by = ["name_for_ordering", "person__name"]
 
-        people_for_post = people_for_post.order_by(
-            F("elected").desc(nulls_last=True), *order_by
-        )
-        people_for_post = people_for_post.order_by(
-            F("votes_cast").desc(nulls_last=True), *order_by
-        )
+        people_for_post = people_for_post.order_by("elected", "-votes_cast", *order_by)
```
Comment (symroe) [inline wcivf/apps/elections/views/mixins.py:99]: This makes sense. I think we need to keep the `F("elected").desc(nulls_last=True)` though, so this line should look like:

```
people_for_post = people_for_post.order_by(F("elected").desc(nulls_last=True), F("votes_cast").desc(nulls_last=True), *order_by)
```

(needs checking)

**Comment by symroe** [comment]
Can you squash the commits down to one? I think this is ok otherwise!

**Comment by symroe** [comment]
Oh and we don't need `by_elected` any more I don't think?

**Review by symroe — APPROVED** [review]

**Pull request merged**